### ISSUE TITLE
Model not finding repeating characters.

### ISSUE THREAD

**Issue by bhi** [arthurflor23/handwritten-text-recognition#20]
Hi,  

I am training the model with 4000 images and the flor architecture.   

I have made the following changes to help with prediction of the model.
max_text_length = 12 
charset_base = string.digits + "."

Here are the results of the training.

[evaluate.txt](https://github.com/arthurflor23/handwritten-text-recognition/files/4376603/evaluate.txt)

[summary.txt](https://github.com/arthurflor23/handwritten-text-recognition/files/4376604/summary.txt)

Here is a example of my images.
![20078 6630](https://user-images.githubusercontent.com/4378481/77454307-58439e00-6dc6-11ea-9e8d-636c75e767ce.jpg)

The problem is when I test the model with any image with repeating characters the repeating characters are consolidated. So the above image the model returns 2078.630

This happens anytime that there is a repeating character and have tested it against several images.
I imagine that this issue wouldn't be so obvious with text data. 

Does anyone have any idea why this is happening?   

Thanks
Danny Haggard
Brown Helicopter Inc

**Comment by arthurflor23** [comment]
Hi, Danny,
This is the default parameter (in low level) of the TensorFlow Beam Search decode (merged=True). TF 1.x had this parameter as option to set, but I didn't see this in 2.x high level api (I don't know why).

An alternative is to use the labels with space between chars: "2 0 7 8 . 6 3 0"

**Comment by bhi** [comment]
Thanks I will give it a go and let you know.

**Comment by bhi** [comment]
Found this issue

https://github.com/keras-team/keras/issues/12238

**Comment by bhi** [comment]
Adding spaces to the labels worked like a charm!  
Thanks For Your Help!

Metrics:
Character Error Rate: 0.95591820
Word Error Rate:      0.99620000
Sequence Error Rate:  1.00000000
